assets/src/edit-story/elements/text/display.js

```diff
@@ -142,14 +128,21 @@ function TextDisplay({
   });
 
   if (backgroundTextMode === BACKGROUND_TEXT_MODE.HIGHLIGHT) {
+    const foregroundContent = content;
+    // Setting the text color of the entire block to black essentially removes all inline
+    // color styling allowing us to apply transparent to all of them.
+    const backgroundContent = useMemo(
```

Comment:
We can't put a hook inside the `if () {}`.

Reply:
Yeah, my bad. Fixed in 13b6fdac.